scripts/pi-hole/php/sidebar.php

```diff
@@ -340,3 +349,15 @@
         </section>
         <!-- /.sidebar -->
     </aside>
+
+<?php
+if (isset($_POST['field'])) {
+    if ($_POST['field'] == 'restartftl') {
+        pihole_execute('-a restartdns');
+    }
+}
+                    ?>
```

Comment:
Why wants `php-cs-fixer` the `?>` so far indented?

Reply:
It doesn't seem to like mixing with HTML, I guess the indentation of
```
                    ?>
```
comes from
```
                <li <?php [...]
```
a few lines above. Interesting that we haven't had such issues during the conversion of the entire codebase.

Reply:
PhpCsFixer indentation is weird.

Sometimes it removes all the indentation from a block, sometimes it adds spaces without a clear motive.